### ISSUE TITLE
Update to 15.0.3

### ISSUE THREAD

**Issue by rasa** [StefanScherer/choco-vmwareworkstation#7]
Per https://docs.vmware.com/en/VMware-Workstation-Pro/15/rn/workstation-1503-release-notes.html, 15.0.3 was released on 14-Mar-2019:
```
What's New
This release of VMware Workstation Pro is a free upgrade for all VMware Workstation 15 Pro users. It contains bug fixes and security updates.

Important Fixes
VMware Workstation Pro contains elevation of privilege vulnerabilities:
Workstation does not handle paths appropriately. This may allow the path to the VMX executable, on a Windows host, to be hijacked by a non-administrator leading to elevation of privilege. The Common Vulnerabilities and Exposures project (cve.mitre.org) has assigned the identifier CVE-2019-5511 to this issue. See VMSA-2019-0002 for further information.
COM classes are not handled appropriately. This may allow hijacking of COM classes used by the VMX process, on a Windows host, leading to elevation of privilege. The Common Vulnerabilities and Exposures project (cve.mitre.org) has assigned the identifier CVE-2019-5512 to this issue. See VMSA-2019-0002 for further information.
Updated OpenSSL library version to 1.0.2q.
Updated libxml2 library version to 2.9.9.

Resolved Issues
Top-level menu options in VMware Workstation Pro are not accessible through  shortcut keys
You are unable to access the VMware Workstation Pro top-level menu items using shortcut key.

**Comment by StefanScherer** [comment]
Fixed with #8, package should be published soon.

**Comment by StefanScherer** [comment]
Thanks for the reminder ;-)
